influxdb3/tests/server/auth.rs

```diff
@@ -82,7 +31,8 @@ async fn auth() {
     );
     assert_eq!(
         client
-            .get("http://127.0.0.1:8181/api/v3/query_sql?db=foo&q=select+*+from+cpu")
+            .get(format!("{base}/api/v3/query_sql"))
```

Comment:
If we're not using the `influxdb3_client` then we could just have these near the top:
```
let query_sql = format!("{base}/api/v3/query_sql");
let write_lp = format!("{base}/api/v3/write_lp");
```
and then do this for all the gets and posts for this test suite.
```suggestion
            .get(&query_sql)
```

get and post should work with &str.